OrbitCore/OrbitSession.h

```diff
@@ -2,28 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#pragma once
+#ifndef ORBIT_CORE_ORBIT_SESSION_H_
+#define ORBIT_CORE_ORBIT_SESSION_H_
 
-#include "Core.h"
-#include "Serialization.h"
+#include "preset.pb.h"
 
-//-----------------------------------------------------------------------------
-struct PresetModule {
-  std::string m_Name;
-  std::vector<uint64_t> m_FunctionHashes;
-
-  ORBIT_SERIALIZABLE;
+struct Preset {
+  std::string file_name;
+  orbit_client_protos::PresetInfo preset_info;
 };
 
-//-----------------------------------------------------------------------------
-class Preset {
- public:
-  Preset();
-  ~Preset();
-
-  ORBIT_SERIALIZABLE;
-
-  std::string m_FileName;
-  std::string m_ProcessFullPath;
-  std::map<std::string, PresetModule> m_Modules;
-};
+#endif  // ORBIT_CORE_ORBIT_SESSION_H_
```

Comment:
Probably a good candidate for protobuf message. @dimitry- what do you think?

Reply:
agree - looks like a proto message to me too.

Reply:
I thought `file_name` is determined at runtime and should not be serialized?

Reply:
It shouldn't be serialized: I thought about having proto message that we don't want to serialize. In this case we will have all preset-related structs in one place.